services-custom/dynamodb-enhanced/src/main/java/software/amazon/awssdk/enhanced/dynamodb/document/DocumentTableSchema.java

```diff
@@ -119,6 +125,9 @@ private List<AttributeConverterProvider> mergeAttributeConverterProviders(Enhanc
         return attributeConverterProviders;
     }
 
+    /**
```

Comment:
nit: `{@inheritDoc}` is only required if you are adding additional information.

Reply:
done